test/system/800-config.bats

```diff
@@ -83,4 +83,45 @@ EOF
     is "$output" "Error:.*no such container"
 }
 
+@test "podman --module - absolute path" {
+    skip_if_remote "--module is not support for remote clients"
+
+    random_data=$(random_string 15)
```

Comment:
In addition to the test failure message, one habit I (inconsistently) have is to add a descriptive prefix to random strings: like containers are `c_$(random_string)`, images `i_`, this could be `="annotation-$(random_string)"`. When looking at failures, just that little bit of a hint makes it easier for my brain to understand what went wrong.